### PR TITLE
* Replace Atlassian icons

### PR DESCRIPTION
**Pull request by vseguin**
image

### CODE REVIEW

File: resources/icons/svg/source-confluence.svg [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (GermainBergeron) [inline resources/icons/svg/source-confluence.svg]: remove `<?xml` tag, comments and ids (e.g., Calque_1, Blue) in every file

File: resources/icons/svg/source-confluence.svg [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
+<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 62.8 60.4" enable-background="new 0 0 62.8 60.4" xml:space="preserve">
```
Comment (GermainBergeron) [inline resources/icons/svg/source-confluence.svg]: You can simplify the svg tag: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 62.8 60.4" enable-background="new 0 0 62.8 60.4">`

File: resources/icons/svg/source-jira.svg [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 73.3 75.8" style="enable-background:new 0 0 73.3 75.8;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#2684FF;}
+	.st1{fill:url(#SVGID_1_);}
+	.st2{fill:url(#SVGID_2_);}
+</style>
+<title>Jira Software-blue</title>
```
Comment (GermainBergeron) [inline resources/icons/svg/source-jira.svg]: Remove the title

File: resources/icons/svg/source-jira.svg [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
-        d="m18.6 18.9c2.8 3.3 2.6 6.9 2.6 6.9h-3.5c.3-1.3-.3-3.2-5.1-7.8-4.7-4.6-4.6-7.7-4.6-7.7 0-.8.7-.4.7-.4 1.3 1 3.1 1 3.1 1 .6 2.4 3.8 5.3 4.3 5.7.7-.7 3.6-3.5 4.1-5.7 0 0 1.8 0 3.1-1 0 0 .7-.4.7.3 0 0 .2 3.3-4.6 7.9-.3.3-.6.5-.8.8m-3.2 3.6c-.6-.8-1.4-1.7-2.5-2.8-2.2 3.1-2 6.2-2 6.2h3.5c-.3-.7-.1-1.7 1-3.4m-1.2-11.3h3.4c0 0 .5-.1.3.5-.2.8-2 2.5-2 2.5s-1.4-1.2-2-2.4c.2 0-.1-.6.3-.6m7.8-3.5c0 .8-.7 1.5-1.5 1.5-.8 0-1.5-.7-1.5-1.5 0-.8.7-1.5 1.5-1.5.9 0 1.5.7 1.5 1.5m-8.8 0c0 .8-.7 1.5-1.5 1.5-.8 0-1.5-.7-1.5-1.5 0-.8.7-1.5 1.5-1.5.8 0 1.5.7 1.5 1.5m4.5-1.2c0 .8-.7 1.5-1.5 1.5-.8 0-1.5-.7-1.5-1.5 0-.8.7-1.5 1.5-1.5.8 0 1.5.7 1.5 1.5"/>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 73.3 75.8" style="enable-background:new 0 0 73.3 75.8;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#2684FF;}
+	.st1{fill:url(#SVGID_1_);}
+	.st2{fill:url(#SVGID_2_);}
+</style>
```
Comment (GermainBergeron) [inline resources/icons/svg/source-jira.svg]: Instead of using classes, put the fill directly on the element

**Comment by vseguin** [comment]
Thank you @GermainBergeron , should be good now!

**Pull request merged**